docs/create_recipe_template.mdx

```diff
@@ -0,0 +1,86 @@
+
+SparseML Recipe Template is built for users who may be inclined to create their recipes by hand to enable more
+fine-grained control or add custom modifiers when sparsifying a new model from scratch with a single API command. 
+
+
+## Overview
+
+SparseML Recipe Template takes in the path to an optional Pytorch model that the you wish to generate a Sparsification recipe for; if none is provided, Recipe Template will generate a generic recipe based on the defaults and designated parameters provided. You may supply a path to a torchscript model (ie *.pt) that can be loaded with `torch.jit.load`. If you are using the python API, you may also supply an already constructed pytorch Module.
+
+
+You have many different options for configuring the Recipe Template API to generate a sparse model to your choosing leveraging techniques like pruning and quantization. Recipe Template has many available arguements which allows you to fully customize the recipe that gets created. A handful of them are required to help designate the techniques used in generating a Sparsification recipe.
+
+To run the SparseML Recipe Template API, you use the following command: 
+
+```bash
+  SparseML.recipe_template(model[optional]) --args[optional]
+```
+
+Arguments can be presented to designate different aspects of the recipe such as whether it should include pruning and if so, what algorithm should be used, quantization and if so, what algorithm should be used, as well as the learning rate type. 
+
+### Base Arguments:
+```bash
+--pruning true|false(default)|ALGORITHM
+ALGORITHM: global magnitude pruning, gradual magnitude pruning – sparsity per layer, ACDC, Movement, MFAC, OBC
+--quantization true|false(default)|TARGET_HARDWARE(default to QAT algorithm ex: vnni, tensorrt)
+--lr TYPE
+TYPE: constant, stepped, cyclic, exponential, linear
+```
+
+### Creating the Recipe Template
+Once SparseML.recipe_template is run, a base recipe template will be created with the necessary top level variables based on the compression methods specified (ie pruning, quantization). The repo will maintain versions for pruning, qat, and pruning+qat.  These templates will either be python strings with `{variable}` placeholders left in for on the fly formatting, or functions that take in the desired variables and return the constructed template.
```

Comment:
```suggestion
Once `sparseml.recipe_template` is run, a base recipe template will be created with the necessary top level variables based on the compression methods specified (ie pruning, quantization). The repo will maintain versions for pruning, qat, and pruning+qat.  These templates will either be python strings with `{variable}` placeholders left in for on the fly formatting, or functions that take in the desired variables and return the constructed template.
```

not too sure what do you mean by maintaining versions

Reply:
I will just get rid of that sentence, if confusing. I took that from the PRD, if it makes no sense let's scrap it.